### PR TITLE
737/calling bot readme

### LINKED ISSUE #737: Request to update bot-calling-meeting tutorial
Hi, I recently followed this tutorial: [Calling and Meeting Bot Sample V4](https://learn.microsoft.com/en-us/samples/officedev/microsoft-teams-samples/officedev-microsoft-teams-samples-bot-calling-meeting-csharp/). And I want to make some fix to it.

1. Typo: the `Get-CsApplicationAccessPolicy -PolicyName Meeting-policy-dev -Identity "<<microsoft-app-id>>"` should actually be `Get-CsApplicationAccessPolicy -Identity Meeting-policy-dev`, according to docs of [Get-CsApplicationAccessPolicy](https://github.com/MicrosoftDocs/office-docs-powershell/blob/main/skype/skype-ps/skype/Get-CsApplicationAccessPolicy.md)
2. I'd like to add a short part for `pwsh` for mac developers, like installing powershell and its modules with homebrew / dotnet. For example <https://learn.microsoft.com/en-us/microsoftteams/teams-powershell-install>
3. folder `TeamsAppManifest` should be `Manifest`(?)
That's all so far. Should I make a PR?

### PR DESCRIPTION
**Pull request by PabloLION**
this PR is a rebased version of #841 and resolves #737 in three major aspects
1. update arguments of CLI [Get-CsApplicationAccessPolicy](https://github.com/MicrosoftDocs/office-docs-powershell/blob/main/skype/skype-ps/skype/Get-CsApplicationAccessPolicy.md)
2. add a link for macOS PowerShell installation. And split the installation of "Microsoft Teams PowerShell Module" from using this module.
3. correct `TeamsAppManifest` folder to `AppManifest`
  - the folder is already renamed, but not reflected in the README.
4. I also made a markdown-lint of the README file. If you don't like it, feel free to revert the last change.

I hope this should meet your standards and can get merged soon because the rebasing for me was quite time-consuming.
@sayali-MSFT @ChetanSharma-msft @Prithvi-MSFT

### CODE REVIEW

**Comment by PabloLION** [comment]
Hi MSFT people @sayali-MSFT @ChetanSharma-msft @Prithvi-MSFT
Can we solve this before the next rebase is needed?

**Review by Pawank-MSFT — APPROVED** [review]
looks correct, Approving!

**Review by Prithvi-MSFT — APPROVED** [review]

**Review by Harikrishnan-MSFT — APPROVED** [review]

**Comment by PabloLION** [comment]
Fantastic! Once we have approval from all reviewers, we're set to proceed with the merge, correct? Are there any additional steps in our workflow that we need to consider before finalizing the process?

**Pull request merged**
